src/test/kotlin/racing/domain/RacingGameTest.kt

```diff
@@ -68,47 +72,8 @@ class RacingGameTest {
             CarState(cars.cars[1].name, CarPosition.INIT_POSITION),
             CarState(cars.cars[2].name, CarPosition.INIT_POSITION)
         )
-    }
-
-    @Test
-    fun `get winner 테스트(winner가 하나)`() {
-
-        val racingGame = RacingGame(testRacingGameData, MovementChecker())
-        val cars = racingGame.cars
-
-        repeat(2) {
-            cars.cars[0].moveCar()
-        }
-
-        repeat(3) {
-            cars.cars[1].moveCar()
-        }
-
-        repeat(1) {
-            cars.cars[2].moveCar()
-        }
-
-        assertThat(racingGame.getWinners().winners).containsOnly(cars.cars[1].name.name)
-    }
-
-    @Test
-    fun `get winner 테스트(winner가 둘)`() {
-
-        val racingGame = RacingGame(testRacingGameData, MovementChecker())
-        val cars = racingGame.cars
-
-        repeat(2) {
-            cars.cars[0].moveCar()
-        }
-
-        repeat(2) {
-            cars.cars[1].moveCar()
-        }
-
-        repeat(1) {
-            cars.cars[2].moveCar()
-        }
 
-        assertThat(racingGame.getWinners().winners).containsOnly(cars.cars[0].name.name, cars.cars[1].name.name)
+        val winners = racingGameResult.winners
+        assertThat(winners.winners).extracting("name").contains("pobi", "crong", "honux")
```

Comment:
winner를 2번 get안하는 방법 찾는다는게 이 정도입니다... 혹시 어떤 방법이 있을까요??

Reply:
`Winners`가 데이터 클래스라는 걸 이용해보세요!!

```suggestion
val expectedWinners = Winners(cars.cars.map { it.name })
assertThat(winners).isEqualTo(expectedWinners)
```